web/modules/custom/genre_validator/genre_validator.module

```diff
@@ -0,0 +1,45 @@
+<?php
+
+/**
+ * @file
+ * This file adds the constraints to the Genre field of the Product CT.
+ *
+ * @category Validator Hook
+ */
+
+declare(strict_types = 1);
+
+use Drupal\Core\Entity\EntityTypeInterface;
+
+/**
+ * Contains hooks and callbacks for genre_validator.module.
+ *
+ * @file
+ */
+
+/**
+ * Parameters used for the hook.
+ *
+ * @param array $fields
+ *   This parameter is used to refer to the genre field.
+ * @param \Drupal\Core\Entity\EntityTypeInterface $entity_type
+ *   This parameter is used to refer to the node entity type.
+ * @param array $bundle
+ *   This parameter is used to refer to the product content type bundle.
+ */
+function genre_validator_entity_bundle_field_info_alter(
```

Comment:
The function's parameters should be on the same line as the function

Reply:
like this? 
`function genre_validator_entity_bundle_field_info_alter(array &$fields, EntityTypeInterface $entity_type, array $bundle): void {`